source/refined-github.css

```diff
@@ -186,3 +186,12 @@ div[data-target='readme-toc.content'] div.blob-wrapper img {
 .ActionListItem .ActionListItem-label--truncate {
 	white-space: unset !important;
 }
+
+/* Add hover effect to attachment row on the release page #8339 */
+/* Test: https://github.com/refined-github/refined-github/releases/tag/25.7.16 */
+.Box-footer .Box--condensed li:has(.octicon-package, .octicon-file-zip):hover {
```

Comment:
This selector (minus `li:hover`) is already used by `release-download-count`, so let's match it